test/testdata/workspaces/python-flask-sample-app/src/main.py

```diff
@@ -2,7 +2,8 @@
 from flask import Flask, jsonify
 
 app = Flask(__name__)
-
+csrf = CSRFProtect()

```

Comment:
Why is this needed?

Reply:
Suggested by SonarQube to enable CSRF protection.